docs/comparisons.md

```diff
@@ -0,0 +1,41 @@
+# Signing Comparisons
+
+## NuGet
+
+The following tables summarize differences between NuGet, dotnet, and Sign CLI's. 
+
+### Features
+
+Feature | NuGet CLI | dotnet CLI | Sign CLI
+-- | -- | -- | --
+Use signing certificate from the file system | ✔️ | ✔️ | ❌
+Use signing certificate from a local store | ✔️ | ✔️ | ❌
+Use signing certificate from Azure Key Vault | ❌ | ❌ | ✔️
+Identify signing certificate by fingerprint | ✔️ | ✔️ | ❌
+Identify signing certificate by subject name | ✔️ | ✔️ | ❌
+Identify signing certificate by name (user-defined) | ❌ | ❌ | ✔️
+Can skip timestamping | ✔️ | ✔️ | ❌
+Opt-in required to overwrite already signed package | ✔️ | ✔️ | ❌
```

Comment:
According to [cli spec draft](https://github.com/dotnet/sign/pull/541/files), `dotnet sign` can overwrite already signed package if I'm reading it correctly.
![image](https://user-images.githubusercontent.com/8766776/211905117-9e092e05-26fd-45f2-b063-f30217b47e3a.png)

Reply:
You are correct, and that's what this doc says.  NuGet and dotnet CLI's allow you to overwrite a signature, but you have to opt in by passing `-overwrite` (NuGet CLI) or `--overwrite` (dotnet CLI).  Overwriting isn't enabled by default for NuGet and dotnet CLI's.  However, Sign CLI overwrites by default with no opt out.

Reply:
If `Sign CLI` overwrites by default, then do we need `-f | --force` option? Or is it for something else?

Reply:
Currently, for scoping reasons, Sign CLI is supposed to match behavior of its predecessor as much as possible.  The old SignService [overwrote signatures by default with no opt-out](https://github.com/dotnet/sign/blob/34f0acefee3673ba192ae0f72e39d029d4d2ef78/src/SignService/SigningTools/NuGetSignService.cs#L126) and [so does Sign CLI](https://github.com/dotnet/sign/blob/b85082906e9c4706efb7580009fe9ebbc320a335/src/Sign.Core/Tools/NuGetSignTool.cs#L45).

This document merely documents current behavior.

I understand and agree with the spirit of a `-overwrite` or `--force` option, but Sign CLI scenarios are more complicated than NuGet's.  With NuGet, you're overwriting a primary signature.  With Sign CLI, you're potentially overwriting different signatures of different types (e.g.:  NuGet, Authenticode, ...).  You may want to _add_ another Authenticode signature but overwrite a NuGet package signature; a simple CLI option won't cut it.